eng/pipelines/runtime-linker-tests.yml

```diff
@@ -0,0 +1,39 @@
+# The purpose of this pipeline is to exercise local developer workflow in the consolidated
+# runtime repo. In particular, it is supposed to run the root "build" script just like any
+# normal developer normally would and monitor regressions w.r.t. this fundamental scenario.
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - release/*.*
+
+pr:
+  branches:
+    include:
+    - master
+    - release/*.*
+
+jobs:
+#
+# Checkout repository
+#
+- template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build Release config vertical for Windows, Linux, Linux musl and OSX
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Windows_NT_x64
+    - OSX_x64
+    - Linux_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Release
+      buildArgs: -s clr+libs -c $(_BuildConfig) /p:LinkerTests=true
```

Comment:
Is the `/p:LinkerTests=true` needed here? Or is that just needed when running them?

Reply:
this is actually running them since we are running the tests during the build, so it is required.

Reply:
But this is running them in the extra steps, not here, right?

Reply:
oh you are right, forgot that we split them, will fix.